src/cmake/ov_parallel.cmake

```diff
@@ -80,8 +80,26 @@ macro(ov_find_package_tbb)
         # conan generates TBBConfig.cmake files, which follows cmake's
         # SameMajorVersion scheme, while TBB itself follows AnyNewerVersion one
         # see https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-version-file
+        if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.22)
+            cmake_host_system_information(RESULT PRETTY_NAME QUERY DISTRIB_PRETTY_NAME)
+        else()
+            execute_process(COMMAND lsb_release -a OUTPUT_VARIABLE output ERROR_QUIET)
+            string(REGEX MATCH "Description:[ \t]+([^\n]+)" match ${output})
+            if(match)
+            set(PRETTY_NAME "${CMAKE_MATCH_1}")
+            endif()
+        endif()
+        set(PKG_CONFIG_SEARCH ON)
         if(CMAKE_TOOLCHAIN_FILE MATCHES "conan_toolchain.cmake" OR CONAN_EXPORTED)
             set(_ov_minimal_tbb_version 2021.0)
+        elseif(${PRETTY_NAME} MATCHES "Ubuntu 22" AND AARCH64)
```

Comment:
should we have something like:
```cmake
if(DISTRO_NAME STREQUAL "Ubuntu" AND DISTRO_VERSION_ID VERSION_GREATER_EQUAL 22.04)
```
But I'm not sure whether Ubuntu 23.04 and 23.10 have TBB 2020.. need to confirm

Reply:
updated